languages/swift/exercises/concept/structs-and-classes/.docs/after.md

```diff
@@ -0,0 +1,298 @@
+
+A summary of what may be changed with structs and classes defined with `let` or `var` is as follows
+
+|         | **struct**                                                                                  | **class**                                                                                  |
+| ------- | ------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
+| **let** | _Cannot_ assign a new struct to the name.</br>_Cannot_ modify any properties of the struct. | _Cannot_ assign a new class to the name.</br>_Can_ modify mutable properties of the class. |
+| **var** | _Can_ assign a new struct to the name.</br>_Can_ modify mutable properties of the struct.   | _Can_ assign a new class to the name.</br>_Can_ modify mutable properties of the class.    |
+
+### Identity
+
+Because two different names may hold a reference to the same instance of class, it is sometimes useful to test if this is the case. Swift uses the `===` _identical_ operator to test for this (along with its negation, `!==`).
+
+```swift
+let char1 = GameCharacter()
+let char2 = char1
+let char3 = GameCharacter()
+
+char1 === char2
+// => true
+char1 === char3
+// => false
+char2 === char3
+// => false
+char2 !== char3
+// => true
+```
+
+Because no two names can refer to the same instance of a struct, these identity operators are not defined for structs.
+
+```struct
+someStats === differentStats
+// Error: Cannot convert value of type 'CharacterStats' to expected argument type 'AnyObject?'
+```
+
+Note that identity is not the same as equality. By many definitions `char1` and `char3` would be considered equal as all of their properties hold the same values, but they do not share the same identity. Identity is made available automatically for all classes in Swift, but developers usually need to write their own equality operators to determine if two instances of a struct or class are _equal_. Doing this is beyond the scope of this exercise.
+
+### Using structs vs. classes
+
+While it may seem as though classes would be favored in general use over structs in Swift as they offer more functionality and aren't burdened with overhead of copying, the opposite is actually true.
+
+Swift, as a language, opts for preferring safety as a core design principle. And classes are harder to reason about with regard to safety, given that their contents can be changed in unexpected ways thanks to being reference types. Because of this Swift prefers the usage of structs to classes and is optimized in many ways with the assumption that structs will be used in most cases. This is why we see that nearly every type in the standard library and in much of Apple's _Foundation_ library are structs.
```

Comment:
I specifically like these 2 paragraphs, they give a good insight into one of Swift's paradigms (saying this from a noob's perspective). ;)